keel-sql/src/main/resources/db/changelog/20200515-add-unique-id-to-event.yml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: add-unique-id-to-event
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: event
+        columns:
+        - name: uid
+          type: varchar(255)
+          constraints:
+            nullable: true
+    - modifySql:
+        dbms: mysql
+        append:
+          value: " first"
+- changeSet:
+    id: add-ref-to-events-with-no-uid
+    author: fletch
+    changes:
+    - sql:
+        sql: >
+          UPDATE EVENT
+          SET JSON = JSON_INSERT(JSON, '$.ref', JSON->'$.id')
+          WHERE NOT(JSON_CONTAINS_PATH(json, 'one', '$.uid'))
```

Comment:
Older existing events had no `uid`.  For those I've synthesized a `ref` from the `id` (which would be the same value)